extensions/wikia/CuratedContent/js/CuratedContentToolButton.js

```diff
@@ -0,0 +1,15 @@
+require(
+	[
+		'jquery'
+	],
+	function ($) {
+		$('#CuratedContentTool').click(function () {
+			var iframe = '<iframe data-url="/main/edit?useskin=mercury" id="CuratedContentToolIframe" name="curated-content-tool" width="100%" height="500"></iframe>',
+				title = 'Mobile Main Page';
+
+			require(['curatedContentTool.modal'], function (curatedContentToolModal) {
```

Comment:
That would mean loading additional file on every page load instead of when user clicks the button. This is a solution copied from WikiaMaps and IMO it makes sense.